.github/workflows/ferretdb_packages.yml

```diff
@@ -149,9 +149,7 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: ${{ matrix.os }}-${{ matrix.arch }}-${{ matrix.pg }}-${{ steps.version.outputs.package_version }}
-          path: |
-            - packaging/*.deb
-            - packaging/*.rpm
+          path: packaging/*.${{ matrix.package }}
```

Comment:
I am puzzled why it didn't work.

This should be same pattern as our last release https://github.com/FerretDB/documentdb/blob/v0.105.0-ferretdb-2.4.0/.github/workflows/ferretdb_packages.yml#L128C17-L128C29